components/rich_text_editor/extensions/emoji/EmojiList.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="d-p4 d-bar8 d-bgc-contrast d-fs-200 d-truncate">
```

Comment:
Not too sure what we should do about this but some of the emoji are hard to read, for example this "TM". A design problem I suppose, doesn't have to be fixed now.

![Screenshot 2023-08-25 at 1 40 10 PM](https://github.com/dialpad/dialtone-vue/assets/64808812/451c54db-13c2-4e33-9eea-dc6d74d2219b)

Reply:
I changed it according to design and its different color. Will push the update